react-visualizer/src/reqJson/klis21-system.json

```
@@ -0,0 +1,313 @@
+{
+    "知識情報・図書館学類21年度入学": {
```

```suggestion
    "知識情報・図書館学類21年度入学(知識情報システム主専攻)": {
```

知識情報システム主専攻以外にも対応しているのであれば無視して問題ないです